### PR TITLE
Startup

### PR DESCRIPTION
**Pull request by natenichols**
### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

### CODE REVIEW

File: dashboard/main.py [hunk 1]
```diff
@@ -2,13 +2,17 @@
 
 import curses
 
-NUM_COLORS = 7 # curses has 8, but white is one of them
+from init import connect
+
+NUM_COLORS = 256 # curses has 8, but white is one of them
```
Comment (mvadari) [inline dashboard/main.py:10]: I thought only some computers supported RGB

File: dashboard/init.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
+import os, curses, time, requests, json
+
+title =  ["""__  ______  ____    _     _____ ____   ____ _____ ____  """,
+          """\ \/ /  _ \|  _ \  | |   | ____|  _ \ / ___| ____|  _ \ """,
+          """ \  /| |_) | |_) | | |   |  _| | | | | |  _|  _| | |_) |""",
+          """ /  \|  _ <|  __/  | |___| |___| |_| | |_| | |___|  _ < """,
+          """/_/\_\_| \_\_|     |_____|_____|____/ \____|_____|_| \_\\"""]
+
+def displayTitle(stdscr, msg):
```
Comment (mvadari) [inline dashboard/init.py]: snake_case pls

File: dashboard/init.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    data = json.dumps({"method": "server_info"})
+    headers = {'content-type': "application/json"}
+    while True:
+        r = None
+        try:
+            r = requests.post("http://" + addr, data=data, headers=headers)
+            print(r.status_code)
+
+        except:
+            None
```
Comment (JST5000) [inline dashboard/init.py]: Instead of having a `None` you can use the `pass` operator (Does nothing :) )

**Review by JST5000 — APPROVED** [review]

File: dashboard/main.py [hunk 4]
```diff
@@ -2,13 +2,17 @@
 
 import curses
 
-NUM_COLORS = 7 # curses has 8, but white is one of them
+from init import connect
```
Comment (mvadari) [inline dashboard/main.py]: This doesn't work for me - I think you need `dashboard.init`

File: dashboard/init.py [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+        elif 'ip' in line:
+            ip = line.split('=')[1].strip()
+
+        if port and ip:
+            return ip + ":" + port
+
+        line = fp.readline()
+
+def findPort():
+    relative = '.config/ripple/rippled.cfg'
```
Comment (mvadari) [inline dashboard/init.py:45]: They might have their config file in a different place - we can fix this later but we should have a .env file for this

Comment (natenichols) [reply dashboard/init.py:45]: This is the default location to store a config (`~/.config/ripple/rippled.cfg`)

**Pull request merged**